legal-api/report-templates/template-parts/amalgamation/completingPartyStmnt.html

```diff
@@ -0,0 +1,38 @@
+<div class="no-page-break">
+    <div class="separator mt-4"></div>
+      <div class="section-title mt-4">
+        <span>Completing Party Statement</span>
+        <div class="section-data mt-3">
+          I,{% for party in amalgamationApplication.parties %} 
+              {% for party_role in party.roles %}
+                {% if party_role.roleType == 'Completing Party' %}
+                  <span class="capitalize-text">{{ party.officer.lastName }}</span>,
+                  <span class="capitalize-text">{{ party.officer.firstName }}</span>
+                  {% if party.officer.middleName %}
+                    <span class="capitalize-text">{{ party.officer.middleName }}</span>
+                  {% endif %}
+                {% endif %}
+              {% endfor %}
+            {% endfor %}, the completing party, have examined the articles and the incorporation agreement applicable to the
```

Comment:
Hopefully there is a more efficient way to write this than the for loops?

Reply:
It should be fine to use for loop as there is reference. https://github.com/bcgov/lear/blob/main/legal-api/report-templates/template-parts/change-of-registration/completingParty.html#L5